tango_simlib/tango_sim_generator.py

```diff
@@ -8,43 +8,34 @@
 Simlib library generic simulator generator utility to be used to generate an actual
 TANGO device that exhibits the behaviour defined in the data description file.
 """
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
+from __future__ import absolute_import, division, print_function
+from future import standard_library
+standard_library.install_aliases()
```

Comment:
@amakhaba This requires a `#noqa` see https://github.com/ska-sa/katcp-python/blob/4240fb45ec8cb73c4b60490e6e43ffe97700f567/doc/conf.py#L17